flow/layers/physical_shape_layer.cc

```diff
@@ -61,57 +59,26 @@ void PhysicalShapeLayer::Preroll(PrerollContext* context,
     set_paint_bounds(path_.getBounds());
   } else {
     if (PhysicalShapeLayerBase::can_system_composite()) {
+      // always system composite these layers when possible and an elevation is
```

Comment:
Comments should be proper prose: https://google.github.io/styleguide/cppguide.html#Punctuation,_Spelling_and_Grammar